gpii/node_modules/contextManager/src/CASConnector.js

```diff
@@ -56,7 +57,8 @@
     };
 
     gpii.contextManager.CASConnector.connectSocket = function (that) {
-        var location =  [2.194909, 41.402455];
+        // var location =  [2.194909, 41.402455];
+        var location =  [0, 0];
```

Comment:
This change didn't seem to be necessary when I tried it out ... I think it depends on what location you give the sensors you're testing with when creating them... Since the example scripts in the CAS use the [2.194909, 41.402455] location, it's probably simplest to stick with them